rna_transcription.py

```diff
@@ -1,11 +1,10 @@
-def to_rna(dna_strand):
-    rna = ''
-    for x in dna_strand:
-        rna = rna + {
+dna_to_rna = {
```

Comment:
@zarniak 👍Minor improvement suggestion - usually, constants' names are all capital letters i.e. `DNA_TO_RNA`. That's to visually distinguish them from plain, mutable variables.